rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/expression/ExpressionGenerator.xtend

```diff
@@ -1109,13 +1100,14 @@ class ExpressionGenerator extends RosettaExpressionSwitch<JavaStatementBuilder,
 	}
```

Comment:
Just something I noticed in above method (`caseToTimeOperation`). The variable `s` should actually be `«lambdaParam»`. (nothing to do with your change)

Reply:
https://github.com/finos/rune-dsl/pull/845/files#diff-fa83dc2efa8118e6ffe7c382e62de2eb5a7b517b846835a05f3332ae78abd3abL1107